scripts/machine-launch-script.sh

```diff
@@ -47,6 +47,12 @@ sudo yum -y install graphviz python-devel
 # used for CI
 sudo yum -y install expect
 
+# Optional: install bloop for fast scala builds on EC2 / CI
+curl -fLo coursier https://git.io/coursier-cli-linux &&
+sudo cp -f coursier /usr/local/bin
+sudo chmod 755 /usr/local/bin/coursier
```

Comment:
I think this is the only unmanaged install to a system-level path? Should we do this?

Reply:
You're right. I was thinking only from the perspective of running on EC2. How would you do it?

Reply:
Put it in `firesim/bloop-install` and put that on the path in `env.sh`?

Reply:
I'm also totally ignoring that this is in machine launch and not the setup script, so maybe just disregard that lol.